Utilities/ci/azure-pipelines.yml

```diff
@@ -64,6 +67,29 @@ jobs:
         poolName: 'Hosted macOS'
         imageName: 'macos-10.13'
         nodeVersion: '12'
+
+  pool:
+    name: '$(poolName)'
+    imageName: '$(imageName)'
+
+  steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '$(nodeVersion).x'
+    displayName: 'Install Node.js'
+
+  - script: |
+      brew install pkg-config pixman cairo pango
+      npm i -g npm@6.4.1
+      npm ci
+      npm run build:release
+    displayName: 'Build'
+
+- job: 'BuildTestWindows'
+  displayName: 'Build and test'
```

Comment:
-> *Build and test* -> *Build* (we are only running the tests on Linux because that is where we have a WebGL environment).